src/commands/Games/coinflip.ts

```diff
@@ -0,0 +1,55 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { MessageEmbed } from 'discord.js';
+import { getColor } from '../../lib/util/util';
+
+enum CoinTypes { Heads, Tails }
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['cf'],
+			bucket: 2,
+			cooldown: 7,
+			description: language => language.tget('COMMAND_COINFLIP_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_COINFLIP_EXTENDED'),
+			requiredPermissions: ['ATTACH_FILES'],
+			runIn: ['text'],
+			usage: '<50|100|200|500|1000|2000|5000|10000> <coin:cointype>',
+			usageDelim: ' '
+		});
+
+		this.createCustomResolver('cointype', (arg, possible, message) => {
+			const coinNames = message.language.tget('COMMAND_COINFLIP_COINNAMES').map(e => e.toLowerCase());
+			if (!coinNames.includes(arg.toLowerCase())) throw message.language.tget('COMMAND_COINFLIP_INVALID_COINNAME', arg);
+			return arg.toLowerCase() === coinNames[0] ? CoinTypes.Heads : CoinTypes.Tails;
```

Comment:
Do the following:
```ts
const lArg = arg.toLowerCase();
const face = message.language.tget('COMMAND_COINFLIP_COINNAMES').find(coin => coin.toLowerCase() === lArg);
if (face === -1) throw message.language.tget('COMMAND_COINFLIP_INVALID_COINNAME', arg);
return face;
```

This works essentially because `CoinTypes.Heads` equals to `0`, and `CoinTypes.Tails` equals to `1`, which is always the case here.